neqo-http3/src/connection.rs

```diff
@@ -1166,6 +1182,34 @@ impl Http3Connection {
                 }
                 Ok(())
             }
+            (Some(s), Some(_r), WebTransportSessionAcceptAction::Redirect) => {
+                if s.http_stream()
+                    .ok_or(Error::InvalidStreamId)?
+                    .send_headers(&[Header::new(":status", "302"), 
+                                  Header::new("location", "/")],
+                                  conn)
+                    .is_ok()
+                {
+                    let extended_conn =
+                        Rc::new(RefCell::new(WebTransportSession::new_with_http_streams(
```

Comment:
I think we don't want to create a WebTransport session, since we already redirect the client to another server.

Reply:
Definitely.  The right answer here is - unfortunately - send this right back to the caller.  We don't do connection management and so we have to bump this entirely.